xcell/tests/test_mapper_k1000.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import xcell as xc
+import healpy as hp
+import os
+
+
+def get_config(w_stars=False):
+    if w_stars:
+        fname = 'xcell/tests/data/catalog_stars.fits'
+    else:
+        fname = 'xcell/tests/data/catalog.fits'
+    return {'data_catalog': fname,
+            'file_nz': 'xcell/tests/data/Nz_DIR_z0.1t0.3.asc',
+            'zbin': 0, 'nside': 32, 'mask_name': 'mask',
+            'e1_flag': 'bias_corrected_e1',
+            'e2_flag': 'bias_corrected_e2'}
+
+
+def get_mapper(w_stars=False):
+    return xc.mappers.MapperKiDS1000(get_config(w_stars))
+
+
+def test_smoke():
+    get_mapper()
+
+
+def get_es():
+    npix = hp.nside2npix(32)
+    return np.repeat(np.array([np.arange(4)]), npix//4,
+                     axis=0).flatten()
+
+
+def test_lite():
+    predir = 'xcell/tests/data/'
+    config = get_config()
+    config['path_lite'] = predir
+    prefix = f'{predir}KiDS1000_lite'
+    os.system(f'rm -r {prefix}*')
+    m = xc.mappers.MapperKiDS1000(config)
+    m.get_catalog()
+    assert os.path.isfile(f'{prefix}_cat_zbin0.fits')
+    m.get_signal_map()
+    assert os.path.isfile(f'{prefix}_shear_e1_ns32_zbin0.fits.gz')
+    assert os.path.isfile(f'{prefix}_shear_e2_ns32_zbin0.fits.gz')
+    assert os.path.isfile(f'{prefix}_galaxies_mask_ns32_zbin0.fits.gz')
+    m.get_nl_coupled()
+    assert os.path.isfile(f'{prefix}_galaxies_w2s2_ns32_zbin0.fits.gz')
+
+    # Non-exsisting fits files - read from lite
+    config['data_catalog'] = 'whatever'
+    m = xc.mappers.MapperKiDS1000(config)
+    m.get_catalog()
+    m.get_mask()
+    m.get_signal_map()
+    m.get_nl_coupled()
+    os.system(f'rm -r {prefix}*')
+
+
+def test_get_signal_map():
+    m = get_mapper()
+    ms = get_mapper(w_stars=True)
+    sh = np.array(m.get_signal_map('shear'))
+    psf = np.array(m.get_signal_map('PSF'))
+    star = np.array(ms.get_signal_map('stars'))
+    es = get_es()
+    assert sh.shape == (2, hp.nside2npix(32))
+    assert psf.shape == (2, hp.nside2npix(32))
+    assert star.shape == (2, hp.nside2npix(32))
+    assert np.all(np.fabs(-sh+(np.mean(es)-es)/(1+m.m[0])) < 1E-5)
+    assert np.all(np.fabs(-psf-es) < 1E-5)
+    assert np.all(np.fabs(-star-es) < 1E-5)
```

Comment:
Wouldn't it be better to use relative differences? (In fact, in this case, doesn't the equality hold?)

Reply:
these are all integers between 0 and 3, so some of them are zero, and you don't want to divide by zero

Reply:
equality is also not a good idea, since these are floats generated with the same operations in different orders

Reply:
make sense